src/PublicKey.h

```diff
@@ -24,6 +24,8 @@ class PublicKey {
     /// The number of bytes in a ed25519 public key.
     static const size_t ed25519Size = 32;
 
+    static const size_t ed25519ExtendedSize = 64;
```

Comment:
why `secp256k1ExtendedSize = 65 ` and `ed25519ExtendedSize ` 64 ? can we be consistent?

Reply:
The Secp extended key format contains a version prefix byte, this is not the case in ED25529 case.  I don't think we can do about this.